tfe/client_mock.go

```diff
@@ -103,3 +103,19 @@ func (m *mockWorkspaces) AssignSSHKey(ctx context.Context, workspaceID string, o
 func (m *mockWorkspaces) UnassignSSHKey(ctx context.Context, workspaceID string) (*tfe.Workspace, error) {
 	panic("not implemented")
 }
+
+func (m *mockWorkspaces) RemoteStateConsumers(ctx context.Context, workspaceID string) (*tfe.WorkspaceList, error) {
```

Comment:
these all need `tfe.*`

Reply:
Fixed! How many times will I have to push a new commit because of fat fingering an error? Only time will tell! 😄